modules/openy_paragraph_activity_finder_search/openy_paragraph_activity_finder_search.info.yml

```
@@ -2,7 +2,7 @@ name: 'OpenY Paragraph Activity Finder Search'
 description: 'Feature with Activity Finder Search paragraph [deprecated].'
 type: module
 package: "YMCA Website Services"
-core_version_requirement: ^8 || ^9 || ^10
+core_version_requirement: ^8 || ^9 || ^10 || ^11
```

```suggestion
core_version_requirement: ^10 || ^11
```

and do this in all info.yml files accross the codebase, please